G/GstPluginsBase/build_tarballs.jl

```diff
@@ -0,0 +1,73 @@
+# the (deprecated...) `T` option to `ar`, until they fix it in Meson.
+# sed -i.bak 's/csrDT/csrD/' build.ninja
+
+ninja -j${nproc}
+ninja install
+install_license gst-plugins-base-*/COPYING
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(exclude=[Platform("i686", "linux", libc = "musl"), Platform("powerpc64le", "linux")])
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libgstpbtypes", :libgstpbtypes, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstoverlaycomposition", :libgstoverlaycomposition, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstaudiotestsrc", :libgstaudiotestsrc, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstaudiomixer", :libgstaudiomixer, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstadder", :libgstadder, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstvideorate", :libgstvideorate, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstvideoconvert", :libgstvideoconvert, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstopus", :libgstopus, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstrawparse", :libgstrawparse, "lib/gstreamer-1.0"),
+    LibraryProduct("libgsttcp", :libgsttcp, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstvolume", :libgstvolume, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstvideotestsrc", :libgstvideotestsrc, "lib/gstreamer-1.0"),
+    LibraryProduct("libgsttypefindfunctions", :libgsttypefindfunctions, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstencoding", :libgstencoding, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstcompositor", :libgstcompositor, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstaudiorate", :libgstaudiorate, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstsubparse", :libgstsubparse, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstaudioconvert", :libgstaudioconvert, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstplayback", :libgstplayback, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstgio", :libgstgio, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstvideoscale", :libgstvideoscale, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstaudioresample", :libgstaudioresample, "lib/gstreamer-1.0"),
+    LibraryProduct("libgstapp", :libgstapp, "lib/gstreamer-1.0"),
+    ExecutableProduct("gst-device-monitor-1.0", :gstdevicemonitor),
+    ExecutableProduct("gst-play-1.0", :gstplay),
+    ExecutableProduct("gst-discoverer-1.0", :gstdiscoverer)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    # Need a host gettext for msgfmt
+    HostBuildDependency("Gettext_jll")
+    Dependency(PackageSpec(name="GStreamer_jll", uuid="aaaaf01e-2457-52c6-9fe8-886f7267d736"); compat="1.20.3")
+    Dependency(PackageSpec(name="Opus_jll", uuid="91d4177d-7536-5919-b921-800302f37372"))
+    # Dependency(PackageSpec(name="ORC_jll", uuid="fb41591b-4dee-5dae-bf56-d83afd04fbc0"))
```

Comment:
Since this was failing to run orcc, maybe this is a [***build*** dependency](https://docs.binarybuilder.org/stable/build_tips/#Dependencies-for-the-target-system-vs-host-system)?